README.md

```diff
@@ -834,6 +834,15 @@ brew install hub
 Rbenv is a tool that we can use to manage multiple versions of Ruby and
 determine which version we use for a particular project.
 
+Before we install Rbenv we want to make sure a similar Ruby manager rvm is not already installed and if it is then we will uninstall it.
```

Comment:
Please limit lines to 80 characters. You can reflow this paragraph in Atom using `cmd + ctrl + Q` or using the action panel.